test/dao_apps.js

```diff
@@ -57,6 +63,35 @@ contract('Applications', accounts => {
       await mockApp.unprotectedDoStuff()
       assert.isTrue(await mockApp.didStuff(), 'should have done stuff')
     })
+
+    it('can create new transaction', async () => {
```

Comment:
We should move all this to its own test file (`integration_accounting` or something like that) as `dao_apps.js` tests that you can install apps in the DAO and not one app in particular.